build/common/installer/scripts/tomlparser.rb

```diff
@@ -191,6 +192,18 @@ def populateSettingValuesFromConfigMap(parsedConfig)
     rescue => errorStr
       ConfigParseErrorLogger.logError("Exception while reading config map settings for adx database name - #{errorStr}, using default #{@adxDatabaseName}, please check config map for errors")
     end
+
+    #Get log loss tracking disablement setting
+    begin
+      if !parsedConfig[:log_collection_settings][:track_lost_logs].nil? && !parsedConfig[:log_collection_settings][:track_lost_logs][:enabled].nil?
+        if !parsedConfig[:log_collection_settings][:track_lost_logs][:enabled].empty?
+           @disableLogLossTracking = parsedConfig[:log_collection_settings][:track_dropped_logs][:enabled]
```

Comment:
what's the difference between track_dropped_logs vs track_lost_logs? i see the top level if checks for track_lost_logs but where as value of track_dropped_logs  being assigned at line # 200?

Reply:
That's an embarrasing bug, fixed.